### ISSUE TITLE
import dependency tree

### ISSUE THREAD

**Issue by mordrax** [stil4m/elm-analyse#105]
I wonder if it's easy to get a tree representation of all the modules that import other modules so we can see when a certain module is changed, which other modules have to recompile.

In our project, we have over 300 files and recompiles are costly bottlenecks of the workflow.

Love your work btw!

**Comment by stil4m** [comment]
Good point! I've been making some structural changes for #102 which already collects all the data for this. In #101 I removed the directed layout while it was mainly missing this information and not properly performing.

I'm thinking about using [http://www.graphviz.org/](http://www.graphviz.org/) but I am not sure yet.

**Comment by mordrax** [comment]
Looks like a nice tool to get some dependency trees going. The current implementation (elm-visualization) has some artefacts where the centre node would jump around.

I didn't realise you started with a directed graph. That information would be awesome to have! I'd happily take on extra CPU cost to see it!

**Comment by stil4m** [comment]
I've released a `digraph` representation that can be viewed using graphviz in the `0.12.0` release. Have you checked it out?
